gatsby-config.js

```diff
@@ -9,6 +9,8 @@ module.exports = {
   },
   plugins: [
     "gatsby-plugin-react-helmet",
+    "gatsby-plugin-ts-checker",
+    "gatsby-plugin-typescript",
```

Comment:
"gatsby-plugin-typescript" does not do typecheck but "gatsby-plugin-ts-checker" does and is more current for type checking for gatsby . An alternative npm package I found is https://github.com/Quramy/typed-css-modules